kitsune/users/tasks.py

```diff
@@ -66,22 +68,14 @@ def process_event_profile_change(event_id):
     event = AccountEvent.objects.get(id=event_id)
     refresh_token = event.profile.fxa_refresh_token
 
-    if not refresh_token:
+    fxa = FXAAuthBackend()
+    token_info = fxa.refresh_access_token(refresh_token)
```

Comment:
since refresh_access_token is a classmethod, would `token_info = FXAAuthBackend.refresh_access_token(refresh_token)` be better?

Reply:
That was my initial approach but since we do need to initialize the class because of the rest of the methods that we are using, I thought to use the same object everywhere to avoid confusion with different calling methods